src/js/tracks/text-track-display.js

```diff
@@ -319,6 +320,45 @@ class TextTrackDisplay extends Component {
       }
       this.updateForTrack(descriptionsTrack);
     }
+
+    if (!window.CSS.supports('inset', '10px')) {
+      const textTrackDisplay = this.el_;
+      const vjsTextTrackCues = textTrackDisplay.querySelectorAll('.vjs-text-track-cue');
+      const controlBarHeight = this.player_.controlBar.el_.getBoundingClientRect().height;
+      const playerHeight = this.player_.el_.getBoundingClientRect().height;
+
+      // Clear inline style before getting actual height of textTrackDisplay
+      textTrackDisplay.style = '';
+
+      // Add custom class to textTrackDisplay div child for not inset support styles
+      textTrackDisplay.firstChild.classList.add('vjs-text-track-display-inset');
```

Comment:
an alternative to adding this class is using css `@supports`. Then, this line won't be necessary and it could all be done in css.
```suggestion
```

Reply:
updated this in the last commit, can this be re-reviewed?